modules/imaging_browser/php/Imaging_Session_ControlPanel.class.inc

```diff
@@ -96,13 +96,17 @@ class Imaging_Session_ControlPanel
             $params['PCandID'] = $timePoint->getCandID();
             $params['PVL']     = $timePoint->getVisitLabel();
         }
-        $tarchiveIDs = $DB->pselect(
-            "SELECT TarchiveID 
-            FROM tarchive 
+
+        // To enable DICOM download: Add pair: tarchiveID, ArchiveLocation
+        $qresult = $DB->pselectRow(
+            "SELECT TarchiveID, ArchiveLocation
+            FROM tarchive
             WHERE PatientName LIKE $ID",
```

Comment:
I confirmed with @AnyhowStep, and he did not see the query the way it is as a security risk.